### ISSUE TITLE
Next Releae

### ISSUE THREAD

**Issue by krishnaguragain** [donjakobo/A3M#103]
When will the next A3M stable version will be released ??

And i am getting this error 

An Error Was Encountered

Unable to load the requested language file: language/english/account/account_profile_lang.php

**Comment by AdwinTrave** [comment]
#116 has updated to CI3 rc2 so check it out now. I think that should fix this issue.

**Comment by AdwinTrave** [comment]
@krishnaguragain are you still getting this error?

**Comment by krishnaguragain** [comment]
this error is fixed but  this debug or error is shown while resetting password.
![reset](https://cloud.githubusercontent.com/assets/6104245/6865177/4d4a2540-d493-11e4-9a8f-6a19d72f7687.PNG)

**Comment by AdwinTrave** [comment]
Yes, that is intentional telling you that you need to tweak your e-mail settings as the current one are not allowing your to send out e-mails.
